BE/api-server/src/main/resources/application-prod.yaml

```
@@ -1,37 +1,37 @@
-      host: ${REDIS_HOST:localhost}
-      port: ${REDIS_PORT:6379}
-      password: ${REDIS_PASSWORD}
-
-server:
-  url: ${SERVER_URL:https://shortlink.example.com}
-  port: 8080
-
-logging:
-  level:
-    io.github.columnwise.shortlink: WARN
+spring:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)